Troubleshooting_Ipv6.md

```diff
@@ -0,0 +1,273 @@
+# IPv6 Disconnected Install Troubleshooting
+
+In this document we will analyze common scenarios of errors/failures which can cuase the cluster installation to fail. 
+
+## Common failures Basic checks before Installing
+
+Below are some precautions we should follow before even attempting the OCP4.3 Ipv6 deployments. 
+
+* [Ipv4-BM-IPI](https://github.com/openshift-kni/baremetal-deploy/blob/master/install-steps.md#preparing-the-provision-node-for-openshift-install) has some pre-requisites which will be applied fpr ipv6 deployments as well. 
+
+### The architecture for IPv6 Disconnected Install is 
+* Jump Host which acts as a DNSMasq, Registry. 
+* Worker/Provisioning node 
+* 3 Master nodes. 
+
+Before starting the installation, we need to set up a DNSmasq conf file which will assign IPv6 IP's to all the nodes on `eno3` interface.A sample file is provided below.  
```

Comment:
Going fwd there be dragons ... We are pushing lab architecture assumptions on to everyone.  This will immediately put us in contention with customer standards.